src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -388,11 +388,20 @@ boolean refersToImpl(T obj) {
     private native boolean refersTo0(Object o);
 
     /**
-     * Clears this reference object.  Invoking this method will not cause this
-     * object to be enqueued.
+     * Clears this reference object. Invoking this method does not enqueue this
+     * object, and the garbage collector will no longer clear or enqueue this
+     * object.
```

Comment:
"will no longer clear" means that it won't do it again, if you see what I mean. I think this would be easier to read if changed to "will not clear ...".